ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -50,6 +50,27 @@ FCU_DFSS_CSP_DXZC:
   - CSP
   - DXZC
 
+FCU_DXRC_RMM:
+  description: "Typical DX FCU with cooling-only (return control) with run mode and failed alarm ."
+  is_canonical: true
+  opt_uses:
+  - failed_alarm
+  - fan_speed_mode
+  implements:
+  - FCU
+  - DXRC
+  - RMM
+  
+FCU_DXASP2:
```

Comment:
see comments on abstract type. I do not know how this device is supposed to control. please explain.